builds/misc/images-release.yaml

```diff
@@ -13,7 +23,7 @@ jobs:
     pool:
         name: $(pool.linux.name)
         demands:
-          - ImageOverride -equals agent-aziotedge-ubuntu-18.04-docker
+          - ImageOverride -equals agent-aziotedge-ubuntu-20.04-docker  
```

Comment:
nit: if you update this review with Bear's suggestions, then please remove the extra space at the end.